### PR TITLE
Review and clarify the sysadmin section about the CLI admin command

### PR DESCRIPTION
**Pull request by sbesson**
- Describes the operations performed by admin start and admin stop
- Mention admin restart and expand the diagnostics command
- Expand the supported options including --force-rewrite

Follows the improvements described in https://github.com/openmicroscopy/openmicroscopy/pull/4379

### CODE REVIEW

File: omero/sysadmins/cli/admin.txt [hunk 1]
```diff
@@ -6,22 +6,91 @@ Server start
...
 
     $ bin/omero admin start
 
-.. note::
+This command performs the following operations in order:
 
-    :omerocmd:`admin start` and :omerocmd:`admin restart` provide a useful
-    debugging and maintenance option called :option:`--foreground`. Using
-    this option allows for starting the server up in the foreground, that is
+#. rewrites the configuration files if :option:`--force-rewrite` is  passed or
```
Comment (mtbc) [inline omero/sysadmins/cli/admin.txt]: double space :scream:
(more below)

File: omero/sysadmins/cli/admin.txt [hunk 2]
```diff
@@ -6,22 +6,91 @@ Server start
...
+   the server has never been started
+#. checks the server status and aborts the command if a server is running
+#. rewrites the configuration files if it has not been done at step 1
+#. starts the server
+#. waits until the server status returns zero
```
Comment (mtbc) [inline omero/sysadmins/cli/admin.txt]: Should it be clearer what the server status is or how to check it?

**Comment by mtbc** [comment]
Staged at http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/cli/admin.html.

File: omero/sysadmins/cli/admin.txt [hunk 3]
```diff
@@ -6,22 +6,95 @@ Server start
...
+#. aborts the command if a server is running
+#. rewrites the configuration files if it has not been done at step 1
+#. starts the server
+#. waits until the server is up
+
+Most configuration files under :file:`etc/grid` are generated using the
+templates under :file:`etc/grid/templates` and the server configuration stored
+in :file:`etc/grid/config.xml`. During this rewriting step, the JVM memory
+settings are recalculated (see :ref:`jvm_memory_settings`) and the new server
+ports (see :ref:`ports_configuration`).
```
Comment (mtbc) [inline omero/sysadmins/cli/admin.txt]: It feels like we're left waiting for the verb re. what happens to the new server ports ("ports" is noun, not verb, right?). Should this be, "during Foo the Bar and Baz are recalculated"? Or do we just need a bit more on the end of the sentence?

File: omero/sysadmins/cli/admin.txt [hunk 4]
```diff
@@ -6,22 +6,95 @@ Server start
...
+
+.. program:: omero admin start
+
+.. option:: -h, --help
+
+    Display the help for this subcommand.
+
+.. option:: --foreground
+
+    This option is particularly useful for debugging and maintenance option
```
Comment (mtbc) [inline omero/sysadmins/cli/admin.txt]: is the second "option" at the end here intended?

**Comment by sbesson** [comment]
@mtbc: pushed.

**Comment by mtbc** [comment]
Good to merge.

**Pull request merged**